django_redshift_backend/base.py

```diff
@@ -461,7 +461,6 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
 redshift_data_types = {
     "AutoField": "integer identity(1, 1)",
     "BigAutoField": "bigint identity(1, 1)",
-    "DateTimeField": "timestamp",
```

Comment:
With this change, the newly created DateTimeField column will be `timestamp with timezone` (TIMESTAMPTZ). Therefore, the existing DateTimeField and the new DateTimeField will have different data types. I think it's not a problem from Django, but database operators will have to always consider the time zone when running SQL directly with `psql`.
@Surgo, do you have any guidelines to avoid this confusion?

Reply:
>  I think it's not a problem from Django

First, Django does not support without `timezone` columns for `psql` users.
See doc: <https://docs.djangoproject.com/en/1.11/topics/i18n/timezones/#postgresql>
When this PR merged, current users have to manually migrate to `with timezone` when set `settings.USE_TIMEZONE = True` or not.
`psql` users are affected too.

So, this is a backward-incompatible change and maybe have to up major version.

Reply:
I understand. Thanks for your explanation. I understood.